### ISSUE TITLE
No sounds show up

### ISSUE THREAD

**Issue by NOTMASTER08** [FalsePattern/SuperSoundMuffler-Legacy#5]
Mixin version: unimixins 0.1.8
FalsePatternLib version: 0.11.3
SuperSoundMuffler-Legacy version: 1.2.0
Relevant log error:
```[15:25:43] [Client thread/ERROR] [Super Sound Muffler: Legacy]: Couldn't access sound registry for sounds list
java.lang.ClassCastException: net.minecraft.client.audio.SoundHandler cannot be cast to com.falsepattern.ssmlegacy.mixin.interfaces.ISoundHandlerMixin
	at com.falsepattern.ssmlegacy.gui.GuiSoundMufflerAddSound.getSoundList(GuiSoundMufflerAddSound.java:78) [GuiSoundMufflerAddSound.class:?]
	at com.falsepattern.ssmlegacy.gui.GuiSoundMufflerAddSound.lazyLoadAllSoundsList(GuiSoundMufflerAddSound.java:71) [GuiSoundMufflerAddSound.class:?]
	at com.falsepattern.ssmlegacy.gui.GuiSoundMufflerAddSound.<init>(GuiSoundMufflerAddSound.java:66) [GuiSoundMufflerAddSound.class:?]
	at com.falsepattern.ssmlegacy.gui.GuiSoundMuffler.func_146284_a(GuiSoundMuffler.java:102) [GuiSoundMuffler.class:?]
	at net.minecraft.client.gui.GuiScreen.func_73864_a(GuiScreen.java:225) [bdw.class:?]
	at net.minecraft.client.gui.inventory.GuiContainer.func_73864_a(GuiContainer.java:326) [bex.class:?]
	at net.minecraft.client.gui.GuiScreen.func_146274_d(GuiScreen.java:296) [bdw.class:?]
	at net.minecraft.client.gui.inventory.GuiContainer.func_146274_d(GuiContainer.java) [bex.class:?]
	at net.minecraft.client.gui.GuiScreen.func_146269_k(GuiScreen.java:268) [bdw.class:?]
	at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:1640) [bao.class:?]
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:973) [bao.class:?]
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:898) [bao.class:?]
	at net.minecraft.client.main.Main.main(SourceFile:148) [Main.class:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_371]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_371]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_371]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_371]
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135) [launchwrapper-1.12.jar:?]
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28) [launchwrapper-1.12.jar:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_371]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_371]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_371]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_371]
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:214) [NewLaunch.jar:?]
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:249) [NewLaunch.jar:?]
	at org.multimc.EntryPoint.listen(EntryPoint.java:143) [NewLaunch.jar:?]
	at org.multimc.EntryPoint.main(EntryPoint.java:34) [NewLaunch.jar:?]```

**Comment by FalsePattern** [comment]
Send the entire log (`fml-client-latest.log`), this exception has *zero* relevant information about mod initialization, mixin injections, etc.

**Comment by NOTMASTER08** [comment]
Ah sorry,i was having trouble with curl and 0x0 and couldn't upload the full logs,i finally managed to fix it.
http://0x0.st/HqJW.log

**Comment by FalsePattern** [comment]
Should be fixed in 1.2.1
